cypress/e2e/spec.cy.ts

```diff
@@ -35,24 +35,20 @@ describe('application user flows', () => {
     cy.get('.nav-item').contains('All Creators');
     cy.get('.nav-item').contains('Saved Creators');
     cy.get('.overlay').should('exist');
-    cy.get('.card').should('have.length', 5);
-    cy.get('.card').first().contains('Mock Aztecross');
-    cy.get('.card').last().contains('Mock Bawkbasoup');
   });
 
   it('takes user to all creators page and displays expected elements by clicking all creators button', () => {
     cy.url().should('eq', 'http://localhost:3000/');
     cy.get('.nav-item').contains('All Creators').click();
+		cy.wait(900)
```

Comment:
Is building in wait time a common practice for Cypress testing?

Reply:
Normally it's not common practice but because we have a fade in animation style for the respective component these elements live in, this buys us a bit of time in the CI environment to grab the element it's expecting without failure in cypress testing for it.